stablehlo/transforms/VhloToVersion.cpp

```diff
@@ -226,12 +238,12 @@ struct VhloToVersionPass : public VhloToVersionPassBase<VhloToVersionPass> {
         [&targetVersion](Operation* op) {
           return isLegalOperation(op, targetVersion);
         });
+    target.addIllegalDialect<stablehlo::StablehloDialect, func::FuncDialect>();
```

Comment:
Why are these dialects called out here?

Reply:
Caught some issues with tests using this one. For the `vhlo-to-version` pass, everything should be VHLO, except builtin.module.